app/controllers/invites_controller.rb

```diff
@@ -100,6 +100,15 @@ def create
     end
   end
 
+  def retrieve
+    params.require(:email)
+
```

Comment:
I recommend we add a guardian method here, like `ensure_can_invite_to_forum` - there is some security built in because it will only retrieve an invite you created, but I would like to ensure you can invite people too.